### PR TITLE
Harmonize headers and pixel polish LLM sync onboarding

### PR DESCRIPTION
**Pull request by thomasrogerlux**
### 📝 Description

Create a new `SyncOnboardingView` component used as a container for all the new onboarding screens, allowing for an unified look and simplification of the code.

### ❓ Context

- **Impacted projects**: `ledger-live-mobile` 
- **Linked resource(s)**: N/A 

### ✅ Checklist

- [ ] ~**Test coverage**~: no UI testing on mobile yet 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

N/A, see proper slack channels

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/screens/SyncOnboarding/ResyncOverlay.tsx [hunk 1]
```diff
@@ -47,12 +48,47 @@ const ResyncOverlay = ({ isOpen, delay = 0, productName }: Props) => {
...
+            <Stop offset="0%" stopOpacity={0} stopColor="transparent" />
+            <Stop
+              offset="10%"
+              stopOpacity={0.4}
+              stopColor={colors.constant.overlay}
+            />
+            <Stop
+              offset="50%"
+              stopOpacity={0.5}
+              stopColor={colors.constant.overlay}
```
Comment (alexandremgo) [inline apps/ledger-live-mobile/src/screens/SyncOnboarding/ResyncOverlay.tsx:75]: same

Comment (alexandremgo) [reply apps/ledger-live-mobile/src/screens/SyncOnboarding/ResyncOverlay.tsx:75]: can you use `"constant.overlay"` ? or get `colors` form `useTheme` if it does not work

**Review by alexandremgo — APPROVED** [review]
:beautiful:

**Comment by abdurrahman-ledger** [comment]
tested and validated with 
https://github.com/LedgerHQ/ledger-live/actions/runs/3243319648
https://github.com/LedgerHQ/ledger-live-build/actions/runs/3248698929

**Pull request merged**